src/GoodList.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+type Props = {
+  good: string;
+};
+
+export class GoodList extends React.PureComponent<Props, {}> {
```

Comment:
The class is called GoodList, but actually it returns `li` that is just list item. I would fix it

Reply:
I have some problems with naming:))